service/src/main/java/bio/terra/drshub/services/DrsResolutionService.java

```diff
@@ -285,8 +285,11 @@ private AccessURL fetchDrsObjectAccessUrl(
             }
             case PASSPORTAUTH -> {
               try {
-                yield drsApi.postAccessURL(
-                    Map.of("passports", auth.orElse(List.of(""))), objectId, accessId);
+                if (auth.isPresent()) {
+                  yield drsApi.postAccessURL(Map.of("passports", auth.get()), objectId, accessId);
+                } else {
+                  yield null;
+                }
```

Comment:
When possible I think it's best to avoid `isPresent()` ... `get()` to avoid duplicating the optional expression. Instead use either `map()` (when producing a value) or `ifPresent()` (when producing side effects only). In this case `map()` should work
```suggestion
                yield auth.map(
                        a -> drsApi.postAccessURL(Map.of("passports", a), objectId, accessId))
                    .orElse(null);
```

Reply:
This looks much nicer, thanks!